source/fuzz/protobufs/spvtoolsfuzz.proto

```diff
@@ -1217,6 +1218,19 @@ message TransformationRecordSynonymousConstants {
 
 }
 
+message TransformationReplaceCopyMemoryWithLoadStore {
+
+   // A transformation that replaces instructions OpCopyMemory with loading
+   // the source variable to an intermediate value and storing this value into the
+   // target variable of the original OpCopyMemory instruction.
+
+   // The intermediate value
+   uint32 source_value = 1;
+
+   // The instruction descriptor
+   InstructionDescriptor instruction_descriptor = 2;
```

Comment:
"instruction_descriptor" is a good name when we want to describe any old instruction.  Here, though, we want to describe an OpCopyMemory, and we're only using an instruction descriptor because OpCopyMemory doesn't have a result id.  Thus I would call this "copy_memory_instruction_descriptor", or just "copy_memory_instruction" to indicate that it is indented to be one of these (I would update the comment a little to reflect this too).